mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json

```diff
@@ -143,6 +164,9 @@
   "bert_ref_12288":
   {
     "Benchmark": "bert",
+    "Creator": "NVidia",
```

Comment:
Actually, technically speaking, it's `NVIDIA` (i.e., all capital letters).

Reply:
Ha, ha. Then your name should be shangw-NVIDIA :)